src/dev/impl/DevToys/ViewModels/Tools/Graphic/ImageConverter/ImageConversionWorkItem.cs

```diff
@@ -0,0 +1,132 @@
+﻿#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+using DevToys.Core.Threading;
+using DevToys.Helpers;
+using DevToys.Shared.Core;
+using DevToys.Shared.Core.Threading;
+using Microsoft.Toolkit.Mvvm.ComponentModel;
+using Microsoft.Toolkit.Mvvm.Input;
+using Windows.Graphics.Imaging;
+using Windows.Storage;
+using Windows.Storage.Pickers;
+using Windows.Storage.Streams;
+
+namespace DevToys.ViewModels.Tools.ImageConverter
+{
+    internal sealed class ImageConversionWorkItem : ObservableRecipient
+    {
+        private string _fileSize = string.Empty;
+
+        internal ImageConverterStrings Strings => LanguageManager.Instance.ImageConverter;
+
+        internal StorageFile File { get; }
+
+        internal string FileSize
+        {
+            get => _fileSize;
+            private set
+            {
+                ThreadHelper.ThrowIfNotOnUIThread();
+                SetProperty(ref _fileSize, value);
+            }
+        }
+
+        internal string? ErrorMessage { get; private set; }
```

Comment:
This property doesn't seem to be used anymore. Can we remove it?

Reply:
Indeed, removed